pkg/clients/instance.go

```diff
@@ -146,9 +149,27 @@ func toIDPortenClientRegistration(in naisiov1.IDPortenClient) types.ClientRegist
 	if in.Spec.PostLogoutRedirectURIs == nil || len(in.Spec.PostLogoutRedirectURIs) == 0 {
 		in.Spec.PostLogoutRedirectURIs = []naisiov1.IDPortenURI{IDPortenDefaultPostLogoutRedirectURI}
 	}
+	if in.Spec.IntegrationType != "" {
+		switch in.Spec.IntegrationType {
+		case string(types.IntegrationTypeIDPorten):
+			if in.Spec.Scopes == nil {
```

Comment:
`in.Spec.Scopes` is never nil. Should instead check for emptyiness here:

```suggestion
			if len(in.Spec.Scopes) == 0 {
```